hooks/useChat.tsx

```diff
@@ -34,7 +35,10 @@ const useChat = () => {
     initialMessages,
     onError: console.error,
     onFinish: async (message) => {
-      await finalCallback(message, messages[messages.length - 2]);
+      await finalCallback(
+        message,
+        messages[messages.length - 2],
```

Comment:
What is `- 2` magic number?
- could you please create a human readable variable name instead of inline `magic numbers`?